projects/step-core/src/lib/modules/editable-labels/components/editable-component/editable.component.ts

```diff
@@ -139,8 +139,12 @@ export abstract class EditableComponent<T> implements ControlValueAccessor {
       return;
     }
 
-    this.value = this.newValue;
-    this.onChange?.(this.value);
+    if (this.onChange) {
```

Comment:
@neogucky @dvladir please check this change. it may affect other existing parts.
Before my change, this component changed the value of the model even if you used only [ngModel]="...".
Now, if (ngModelChange) is present, it is responsible to assign the value (this gives us control over the 'apply' or 'cancel' buttons. e.g if some validations are not respected, on 'Apply' click, the value can be reverted to the initial value.